admin/src/scenes/settings/tabs/EligibilityTab.tsx

```diff
@@ -0,0 +1,223 @@
+              onChange={(e) => setFormValues({ ...formValues, bornAfter: e })}
+              error={error?.bornAfter || ""}
+              readOnly={false}
+              disabled={isLoading || readOnly}
+            />
+            <DatePickerInput
+              className={""}
+              mode="single"
+              label="Né avant le"
+              placeholder="JJ/MM/AAAA"
+              isTime={false}
+              value={formValues.bornBefore}
+              onChange={(e) => setFormValues({ ...formValues, bornBefore: e })}
+              error={error.bornBefore || ""}
+              readOnly={false}
+              disabled={isLoading || readOnly}
+            />
+          </div>
+        </div>
+        {error?.interval && <div className="text-[#EF4444]">{error?.interval}</div>}
+      </div>
+      <div className="flex flex-col gap-8 rounded-xl bg-white px-8 pb-12 pt-8 shadow-[0_8px_16px_0_rgba(0,0,0,0.05)]">
+        <div className="flex w-full flex-col gap-8">
+          <p className="text-lg font-medium leading-5 text-gray-900">Éligibilités par situations scolaire</p>
+          <div className="flex row w-full gap-4 justify-center">
+            {Object.keys(GRADES).map((grade, index) => (
+              <div key={grade + "grade" + index} className="flex h-6 items-center">
+                <input
+                  id={grade}
+                  name={grade}
+                  type="checkbox"
+                  disabled={isLoading || readOnly}
+                  checked={formValues.schoolLevels.includes(grade)}
+                  className="h-4 w-4 rounded border-gray-300 text-blue-600 cursor-pointer"
+                  onChange={(e) => {
+                    const newValues = e.target.checked ? [...formValues.schoolLevels, grade] : formValues.schoolLevels.filter((g) => g !== grade);
+                    setFormValues((prev: EligibilityForm) => ({ ...prev, schoolLevels: newValues }));
+                  }}
+                />
+                <label htmlFor={grade} className="mt-1.5 ml-2 text-sm cursor-pointer whitespace-nowrap">
+                  {translateGrade(grade)}
+                </label>
+              </div>
+            ))}
+          </div>
+          {error.schoolLevels && <div className="text-[#EF4444]">{error.schoolLevels}</div>}
+        </div>
+      </div>
+      <div className="flex flex-col gap-8 rounded-xl bg-white px-8 pb-12 pt-8 shadow-[0_8px_16px_0_rgba(0,0,0,0.05)]">
+        <div className="flex w-full flex-col gap-8">
+          <p className="text-lg font-medium leading-5 text-gray-900">Éligibilités par départements</p>
+
+          <div className="grid grid-cols-2 gap-10">
+            {Object.keys(region2department).map((region, index) => {
+              const departmentsSelected = formValues?.zones?.filter((dep) => region2department[region].includes(dep));
+              const allDepartmentSelected = departmentsSelected.length === region2department[region].length;
```

Comment:
```suggestion
              const isAllSelected = departmentsSelected.length === region2department[region].length;
```
si c'est un boolean, on peux préfixé par "is" ou "has" pour une meilleur lisibilité.